inform7/Internal/Inter/Architecture32Kit/Sections/Input Output.i6t

```diff
@@ -278,7 +278,9 @@ to document all of that.
     return key;
 ];
 
-[ VM_ReadKeyboard  a_buffer a_table done ix;
+Array UnicodeWhitespace --> 8 133 160 5760 8232 8233 8239 8287 12288;
```

Comment:
I guess the 8 should be removed now that all code points < 32 get are handled without looking at this array?

Reply:
Nevermind, this is the number of entries. I got confused looking at the previous version of the code.